src/index.js

```diff
@@ -10,4 +10,12 @@ Reflux.listenTo = require('./listenTo');
 
 Reflux.listenToMany = require('./listenToMany');
 
+/* globals React: false */
+Reflux.defineReact = require('./defineReact');
+try {
+	if (React) {
+		Reflux.defineReact(React, Reflux);
+	}
+} catch (e) { }
```

Comment:
Oh ok, makes sense. Since errors are being dropped it would be good to isolate the try/catch to only the check for React's existence so that it doesn't accidentally drop any errors from the call to defineReact(). The only thing worse than getting an error is an error that you don't get ;).

So the check becomes...

`if (reactExists()) {
    Reflux.defineReact(React, Reflux);
}`

or...

`var react = safelyGetReact()
if (react) {
    Reflux.defineReact(react, Reflux);
}`

...and then the try/catch is only used for checking existence:

`function reactExists() {
    try {
        if (React) {
            return true;
        }
    } catch() {}

    return false;
}`